e2e/package-lock.json

```diff
@@ -12,9 +12,7 @@
         "@axe-core/playwright": "4.8.5",
         "@percy/cli": "1.27.4",
         "@percy/playwright": "1.0.4",
```

Comment:
i think this might have got picked up incorrectly

Reply:
wym?

Reply:
@ozyx @akhenry broke this out into a separate discussion https://github.com/nasa/openmct/issues/7847